src/main/java/com/exactpro/th2/conn/ServiceListener.java

```diff
@@ -113,7 +114,9 @@ public void sessionIdle(IServiceProxy service, IdleStatus status) {
     public void exceptionCaught(IServiceProxy service, Throwable cause) {
         LOGGER.error("Session '{}' threw exception", sessionAlias, cause);
         try {
-            Event event = Event.start().endTimestamp()
+            Event event = Event.start()
+                    .bookName(idBuilder.get().getBookName())
```

Comment:
Please avoid creating a new builder for taking "static" values.
Let's pass or get it only once in the constructor

Reply:
done